eugl/gqa/tasks.py

```diff
@@ -55,17 +71,455 @@
 # rather than find and read the internal cfg.
 # Similar to how wagl and tesp do it
 CONFIG = luigi.configuration.get_config()
-CONFIG.add_config_path(resource_filename('gqa', 'gqa-defaults.luigi.cfg'))
+
+# TODO purge this
+# CONFIG.add_config_path(resource_filename('gqa', 'gqa-defaults.luigi.cfg'))
```

Comment:
Is this purgeable now or later?

Reply:
Since we are going to refactor it to work with Landsat anyway, now. Done.